Application/EdFi.Ods.Api.IntegrationTestHarness/Program.cs

```diff
@@ -29,31 +29,15 @@ public static async Task Main(string[] args)
                 .ConfigureAppConfiguration(
                     (hostingContext, config) =>
                     {
-                        var env = hostingContext.HostingEnvironment;
-
                         config
                             .SetBasePath(Directory.GetParent(AppContext.BaseDirectory).FullName)
```

Comment:
Should this be AppContext.BaseDirectory instead?

Reply:
our intent was to remove this and use the default builder